wormhole-connect/src/views/Redeem/DestinationQueued.tsx

```diff
@@ -0,0 +1,173 @@
+import InputContainer from 'components/InputContainer';
```

Comment:
we should make sure @emreboga is aware of this view being added to the legacy UI. I don't know if the redesign covered any of these edge cases with queues/governor (probably not)